test/libraries/EscrowBits.t.sol

```diff
@@ -28,7 +24,7 @@ contract EscrowBitsTest is Test {
         );
 
         expectedBitMapString =
-            "0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000011111100000000000000000";
+            "0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000111011100000000000000000";
```

Comment:
I don't think we should have SolverOutcome arranged such that there's a 0 in the middle of the 1s like there is here.  Not sure what's happening to cause this

Reply:
Resoved